### PR TITLE
Added notes for pnpm users when deploying to Netlify

### PR DESCRIPTION
**Pull request by RafidMuhymin**
Netlify requires additional settings if using `pnpm`. This PR adds notes for `pnpm` users when deploying to Netlify.

### CODE REVIEW

**Review by delucis — COMMENTED** [review]
Looks good @RafidMuhymin. How about including that link you shared in Discord to the Netlify pnpm guide?

**Comment by johnnytest4real** [comment]
[Credit to this person](https://community.cloudflare.com/t/add-pnpm-to-pre-installed-cloudflare-pages-tools/288514/5) for linking the fix, which actually came from [Vite's Netlify config](https://github.com/vitejs/vite/blob/155fd11b783eddd33f0cd7e411eea40a3585217a/netlify.toml).

Relatedly, I'm wondering if there's a similar config that can be created for other hosts that don't fully support pnpm (e.g., Cloudflare Pages).

File: src/pages/en/guides/deploy.md [hunk 1]
```diff
@@ -213,6 +213,16 @@ Create a new `netlify.toml` file at the top level of your project repository wit
   publish = "dist"
 ```
 
+If you are using `pnpm`, then set the following settings instead:
```
Comment (delucis) [inline src/pages/en/guides/deploy.md:216]:
```suggestion
Using `pnpm`? [Netlify needs some extra set-up to support `pnpm`.](https://answers.netlify.com/t/using-pnpm-and-pnpm-workspaces/2759) We suggest starting with the following settings in your `netlify.toml`:
```

**Review by delucis — COMMENTED** [review]
Just to follow up my previous comment, I think adding a link like this could be helpful to let people dive deeper if they need to.

File: src/pages/en/guides/deploy.md (hunk 1 above)
Comment (sarah11918) [reply src/pages/en/guides/deploy.md:216]:
```suggestion
Using [`pnpm` with Netlify?](https://answers.netlify.com/t/using-pnpm-and-pnpm-workspaces/2759) Use the following settings instead:
```

Comment (delucis) [reply src/pages/en/guides/deploy.md:216]: ✂️💨 Scissors to the rescue! LGTM.

**Comment by sarah11918** [comment]
The change above is also good for SEO. What do you y'all think?

Othewise, @RafidMuhymin LGTM! Merge when you're happy with it!

File: src/pages/en/guides/deploy.md (hunk 1 above)
Comment (delucis) [reply src/pages/en/guides/deploy.md:216]: Ah, good question. I think I’d say “on”?

Comment (sarah11918) [reply src/pages/en/guides/deploy.md:216]: Updated in my suggestion, thanks! @RafidMuhymin can make the final call.  :)

**Pull request merged**